deploy/eck-integrations/examples/multiple-integrations/README.md

```diff
@@ -0,0 +1,54 @@
+# Example: Multiple Integrations
+
+In this example we install the built-in `kubernetes` integration and a `nginx` custom integration bound on the same agent cluster-wide preset that `kubernetes` integration utilises. Also, we utilise the hints-based autodiscovery of integration supported in `kubernetes` integration. 
+
+## Prerequisites:
+1. Installed eck-operator helm chart
+   ```console
+   helm repo add elastic https://helm.elastic.co && helm repo update
+   helm install elastic-operator elastic/eck-operator --create-namespace
+   ```
+2. For **non** eck-managed ElasticSearch clusters you need a k8s secret that contains the connection details to it such as:
+    ```console
+    kubectl create secret generic es-ref-secret \
+        --from-literal=username=... \
+        --from-literal=password=... \
+        --from-literal=url=...
+    ```
+    Note: specifying an `api-key`, instead of a `username` and `password`, is not supported at the moment but there is an already open PR to add support for it.
+
+3. `kubernetes` integration Assets are installed through Kibana
+4. `redis` integration Assets are installed through Kibana
+5. `nginx` integration Assets are installed through Kibana
```

Comment:
```suggestion
4. `redis` integration assets are installed through Kibana
5. `nginx` integration assets are installed through Kibana
```
how?

Reply:
thx for the question. Added links that show how to achieve this through Kibana 🙂